### ISSUE TITLE
Fine Tuning Did Not Decreased DER 

### ISSUE THREAD

**Issue by goncatoprak** [pyannote/pyannote-audio#1384]
Hi,

I was trying to fine-tune the pyannote with the AISHELL-4 dataset (https://www.openslr.org/111/) via the tutorial you shared (adapting_pretrained_pipeline.ipynb). I downloaded train_S, train_L, train_M and test sets of AISHELL-f and shuffle themThe train set was 80%, test and dev sets were 10% of this shuffled AISHELL-4 data. Then, I tested the fine-tuned model on my private dataset. During the training, I did not get any errors but the learning curves did not change at all. Therefore, there was no improvement regarding DER values. The base pyannote model's DER value on my test set was 42.8%.

If I try to fine-tune pyannote with the whole AMI corpus, will I see any improvements, or decreases in DER values for my test set? Do you think that the reason was that I tried to fine-tune with AISHELL-4 rather than AMI corpus? Do you have any suggestions?

Thanks.
